assets/dev/js/editor/elements/views/behaviors/context-menu.js

```diff
@@ -5,6 +5,7 @@ var ContextMenu = require( 'elementor-editor-utils/context-menu' );
 module.exports = Marionette.Behavior.extend( {
 
 	defaults: {
+		isIframe: true,
```

Comment:
Consider adding docs to explain this property usage.

Reply:
`context: 'preview'`